lib/src/components/NxGlobalSidebar/NxGlobalSidebar.tsx

```diff
@@ -47,18 +49,22 @@ const NxGlobalSidebar: FunctionComponent<Props> = function NxGlobalSidebar(props
     </NxButton>
   );
 
-  const logo = <img src={logoImg} alt={logoAltText} className="nx-global-sidebar__logo" />;
+  const logo = <img src={logoImg} alt={logoAltText} className="nx-global-sidebar__logo" />,
+      [footer, otherChildren] = splitOutFirst(NxGlobalSidebarFooter, children);
 
   return (
-    <aside className={classes} id={id} aria-label="global sidebar">
-      <div className="nx-global-sidebar__header">
-        <a className="nx-global-sidebar__product-info nx-global-sidebar__expanded-content"
-           href={logoLink}>
-          { logo }
-        </a>
-        { toggleButton }
+    <aside className={classes} id={id} aria-label="global sidebar" role="group">
+      <div role="complementary">
```

Comment:
If you make this the `aside` and the other element a `div` then you don't need to be explicit with the role here.

Reply:
> If you make this the `aside` and the other element a `div` then you don't need to be explicit with the role here.

Changed here: bc41d7e723b1445e48ec2ec12229e666e5bc4aec